api-specificatie/openapi.yaml

```diff
@@ -475,6 +483,7 @@ paths:
       operationId: raadpleegPand
       tags:
         - Pand
+      summary: "levert een pand"
```

Comment:
Is het een idee om hier van te maken: "levert details van een <object>" en dit ook door te voeren bij de andere objecten? Dan is het consistent.

Reply:
@strijm het is juist de bedoeling om een onderscheid te maken tussen de endpoints met adressen, panden en ADO's en de "detail"endpoints nummeraanduiding, woonplaats en openbare ruimte.